### PR TITLE
Test Dart 2.18

### PR DESCRIPTION
**Pull request by sourcegraph-wk**
Summary
---
This batch overrides the docker images used to equivalent Dart 2.18 versions.
The goal is to run as many CI runs across the dart ecosystem as possible on 
Dart 2.18 to find and fix errors before releasing a production 2.18 image.

For more info, reach out to `#support-frontend-dx` on Slack.

[_Created by Sourcegraph batch change `Workiva/test_dart_218`._](https://sourcegraph.plat.workiva.net/organizations/Workiva/batch-changes/test_dart_218)

### CODE REVIEW

**Review by greglittlefield-wf — DISMISSED** [review]

File: pubspec.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ dependencies:
   w_common: '^2.0.0'
 
 dev_dependencies:
-    dart_dev: ^3.8.7
+    build_runner:
+    build_web_compilers:
```
[the remark this answers is not in the document]
Comment (robbecker-wf) [reply pubspec.yaml:15]: we really only need them if we want to be able to `webdev serve example:8080` as it checks to see if you have these as dependencies.

[the remark this answers is not in the document]
Comment (robbecker-wf) [reply pubspec.yaml]: So, yes they are required.

**Review by evanweible-wf — APPROVED** [review]

**Comment by robbecker-wf** [comment]
QA+1 CI Passes on 2.13 and 2.18 @Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**